packages/alchemy/src/react/components/auth/card/main.tsx

```diff
@@ -5,13 +5,11 @@ import { AuthSection } from "../sections/AuthSection.js";
 import type { AuthCardProps } from "./index.js";
 
 // eslint-disable-next-line jsdoc/require-jsdoc
-export const MainAuthContent = ({
-  header = "Sign in",
-  sections,
-}: AuthCardProps) => {
+export const MainAuthContent = ({ header = null, sections }: AuthCardProps) => {
   return (
     <>
-      <h3 className="font-bold text-lg">{header}</h3>
+      {header}
+      <h3 className="font-bold text-lg">Sign in</h3>
```

Comment:
I'm wondering if the `Sign in` should probably be configurable as well here. Some devs might not want to render `Sign in` and so it's a worthwhile escape hatch

Reply:
Oh I was going off of the Slack discussion, let's pick it up there - it feels to me like if the sign in header text is configurable, then I'd expect other headers to be? But maybe we just have a boolean prop to hide it?